src/templates/podcast.js

```diff
@@ -83,6 +80,29 @@ const StyledPlayer = styled(AudioPlayer)`
     }
 `;
 
+const StyledBlogShareLinks = styled(BlogShareLinks)`
+    flex-direction: column;
+    align-items: center;
+    > :not(:first-child) {
+        margin-top: ${size.medium};
```

Comment:
<img width="236" alt="Screenshot 2021-05-17 at 23 58 28" src="https://user-images.githubusercontent.com/79590294/118555812-c9cd7100-b76b-11eb-9575-4af64deb31b2.png">
Also, here we need this margins only for screen 1024px and hire.